cluster-autoscaler/cloudprovider/clusterapi/clusterapi_utils_test.go

```diff
@@ -117,18 +117,18 @@ func TestUtilParseScalingBounds(t *testing.T) {
 	}, {
 		description: "deprecated min/max annotations still work, result is min 0, max 1",
```

Comment:
could we change these test descriptions to say "cluster-api min/max annotations", or something similar